package.json

```diff
@@ -10,7 +10,8 @@
     "npm": ">=7.6.0"
   },
   "dependencies": {
-    "fs-extra": "9.0.1"
+    "fs-extra": "9.0.1",
+    "svelte-awesome": "^2.3.2"
```

Comment:
Do not use svelte awesome,  use svelte carbon icons so that the icons stays coherents

Reply:
The font awesome code was just a copy/paste legacy from hakuneko :)

Reply:
Ok got it :)